cajita/unit_test/tstHypreStructuredSolver2d.hpp

```diff
@@ -203,52 +203,72 @@ poissonTest( const std::string& solver_type, const std::string& precond_type,
 //---------------------------------------------------------------------------//
 TEST( structured_solver, pcg_none_test )
 {
+    HYPRE_Init();
     poissonTest( "PCG", "none", TEST_MEMSPACE{} );
+    HYPRE_Finalize();
```

Comment:
Did you consider doing it in `main()` instead?

Reply:
I agree - a single new scope within the poisson function would be cleaner